gslib/utils/shim_util.py

```diff
@@ -158,6 +200,26 @@ def _get_gcloud_binary_path():
   return os.path.join(cloudsdk_root, 'bin', 'gcloud')
 
 
+def _get_gcs_json_endpoint_from_boto_config(config):
+  gs_json_host = config.get('Credentials', 'gs_json_host')
+  if gs_json_host:
+    gs_json_port = config.get('Credentials', 'gs_json_port')
+    port = ':' + gs_json_port if gs_json_port else ''
```

Comment:
Could this result in an extra `:` if no port?
Should it be:
`port = (':' + gs_json_port) if gs_json_port else ''`

Reply:
No, everything before the `if` will be evaluated if the condition is true.